Dockerfile.windows

```diff
@@ -283,5 +284,8 @@ ENTRYPOINT ["powershell.exe"]
 # Set the working directory to the location of the sources
 WORKDIR ${GOPATH}\src\github.com\docker\docker
 
+ARG WINDOWS_BASE_IMAGE_TAG
+ENV WINDOWS_BASE_IMAGE_TAG=${WINDOWS_BASE_IMAGE_TAG}
```

Comment:
Given the testing code appears to read `WINDOWS_BASE_IMAGE`, we probably want both here, right?